test/cypress/plugins/index.js

```diff
@@ -0,0 +1,19 @@
+// ***********************************************************
```

Comment:
I think this is an example file that we don't need?

Reply:
I'm going to leave it for instructional purposes for future test authors. I don't think it's hurting anything, since it's not included in our bundles.

Reply:
This file is also expected by Cypress, from what I can tell.